yapapi/network.py

```diff
@@ -54,6 +55,26 @@ def get_websocket_uri(self, port: int) -> str:
         return f"{net_api_ws}/net/{self.network.network_id}/tcp/{self.ip}/{port}"
 
 
+class NetworkStateMachine(StateMachine):
```

Comment:
I'd call this class `NetworkState` for a bit more brevity and to make it consistent with `ServiceState`

```suggestion
class NetworkState(StateMachine):
```

Reply:
Changed in c5d425c